src/modules/cloudflare/captcha/types.ts

```diff
@@ -0,0 +1,12 @@
+export interface ICaptchaProtectedReturn{
+    success: boolean
+    message: string,
+    data: {
+        token: string
+    }
+}
+export interface ITokenResponse{
+    body: {
+        token: string
+    }
+}
```

Comment:
We didint use the prefix I on the project, please could you remove this ? -> TurnstileVerificationResponse

Reply:
prefix removed